### ISSUE TITLE
"Wide review" is too easy to confuse with "horizontal review"

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/w3process#776]
It might be better if we could come up with something more clear, like "public review" or even "public-wide review", to more clearly capture its intent and broad scope? (and to distinguish it more clearly from the undefined, yet, often used 'horizontal review').

**Comment by samuelweiler** [comment]
FWIW, [How to do Wide Review](https://www.w3.org/Guide/documentreview/) says "Much of this document focuses on how and when to conduct horizontal reviews, but they are only a subset of a full wide review, which must also include other stakeholders..."

@nigelmegitt has an open issue on that document saying "[Title is about Wide Review, contents are about Horizontal Review](https://github.com/w3c/documentreview/issues/12)"

**Comment by plehegar** [comment]
[[
tzviya: I think this is one of the confusing points of the process. People generally use the terms interchangeably even though we know they're not. We should clarify.

fantasai: I think this is largely a problem with how we have been educating people on these terms. E.g. the document on wide review is 95% on horizontal review.
… the part that's outside horizontal review gets lost because we don't talk about it, and it doesn't have formal structure.

<plh>6.2.2.1. Wide Review

florian: I agree there is confusion, Elika's hypothesis on why is plausible. Maybe we should start there.
… as to Tzviya's point about simplification, I agree, but we should not talk about process simplification in general, we need to be specific. Maybe this is one of those specific points, maybe it isn't. We should be detailed.

<fantasai> [w3c/documentreview#12](https://github.com/w3c/documentreview/issues/12)

plh: I don't hear anyone saying the process is broken...
… we could decide to push this to the guidebook first? it's not clear this is an actual process issue.

florian: maybe we should start there, and if it becomes obvious there is a process problem here, we circle back
… absent a specific idea, I would start with the Guide.

plh: the proposal is to move this to the Guide.
… this will be posted there unless someone reacts within a week
]]
https://www.w3.org/2023/09/27-w3process-minutes.html#t04

**Comment by dwsinger** [comment]
perhaps we need to clarify that wide review has at least these components:
* horizontal review by groups with a specific area of horizontal concern (notably acessibility, i18n, and architecture/TAG)
* an explicit opportunity and request for consortium review (membership, team and IEs)
* an opportunity and request for public review

**Comment by nigelmegitt** [comment]
> an explicit opportunity and request for consortium review (membership, team and IEs)

Do we request consortium review? I know we explicitly request review from W3C groups that are listed in the publishing WG's Charter, and there is probably some generic automated email that goes to some wide groups of people saying "new FPWD" or "patent exclusion opportunity", but I don't count those generic automated emails as being effective tools for requesting consortium review - they're either being piped to legal teams or dropped to the bottom of the priority pile of busy people.

It'd probably be a good idea though, because when we do finally get around to a proper consortium review, which is AC review of CR exit, that is uncomfortably late to receive negative comments.

**Comment by dwsinger** [comment]
Yes, an AC vote is the formal review by the consortium members. yes, getting those review comments earlier would be better

**Comment by nigelmegitt** [comment]
Right - the Process requires Wide Review for _entering_ CR, but that definitely does not require AC review.
